File: php/loop.php
<?php
/*
 * Copyright (c) 2024.
 * Saman beheshtian
 * amintado@gmail.com
 * +989353466620
 * https://yiiman.ir
 */


$end_number=1000000000;
;
$start_time=microtime(true);

for ($i=1; $i<=$end_number; $i++){
    if ($i==$end_number){
        echo 'End after: '.(microtime(true)-$start_time);
    }
}

File: go/loop.go
package main

import (
    "fmt"
    "time"
)

func main() {
    endNumber := 1000000000
    startTime := time.Now()

    for i := 1; i <= endNumber; i++ {
        if i == endNumber {
            elapsed := time.Since(startTime)
            fmt.Printf("End after %v milliseconds\n", elapsed.Milliseconds())
        }
    }
}

